src/assets/translations/messages-en.json

```diff
@@ -1,344 +1,350 @@
-      "invalidAddress": "Invalid Address",
-      "deposit": "Deposit",
-      "withdraw": "Withdraw",
-      "getStarted": "Get Started",
-      "connectWallet": "Connect Wallet",
-      "seeAll": "See All"
+  "common": {
+    "back": "Back",
+    "next": "Next",
+    "error": "Error",
+    "cancel": "Cancel",
+    "continue": "Continue",
+    "confirm": "Confirm",
+    "reject": "Reject",
+    "noResultsFound": "No results found.",
+    "insufficientFunds": "Insufficient Funds",
+    "invalidAddress": "Invalid Address",
+    "deposit": "Deposit",
+    "withdraw": "Withdraw",
+    "getStarted": "Get Started",
+    "connectWallet": "Connect Wallet",
+    "seeAll": "See All"
+  },
+  "dashboard": {
+    "portfolio": {
+      "addAsset": "Add Asset",
+      "yourAssets": "Your Assets",
+      "asset": "Asset",
+      "portfolioBalance": "Portfolio Balance",
+      "balance": "Balance",
+      "price": "Price",
+      "allocation": "Allocation"
     },
-    "dashboard": {
-      "portfolio": {
-        "addAsset": "Add Asset",
-        "yourAssets": "Your Assets",
-        "asset": "Asset",
-        "portfolioBalance": "Portfolio Balance",
-        "balance": "Balance",
-        "price": "Price",
-        "allocation": "Allocation"
+    "recentTransactions": {
+      "recentTransactions": "Recent Transactions"
+    }
+  },
+  "earn": {
+    "emptyVaults": "Vaults will appear here",
+    "yearn": {
+      "somethingWentWrong": "Something went wrong",
+      "somethingWentWrongBody": "Something went wrong",
+      "transactionFailed": "Transaction Failed",
+      "transactionFailedBody": "Transaction Failed"
+    }
+  },
+  "navBar": {
+    "dashboard": "Dashboard",
+    "assets": "Assets",
+    "earn": "Earn",
+    "tradeHistory": "Trade History",
+    "foxToken": "FOX Token"
+  },
+  "transactionRow": {
+    "send": "Sent",
+    "receive": "Received",
+    "moreDetails": "More Details",
+    "date": "Date",
+    "txid": "Transaction ID",
+    "fee": "Fee",
+    "status": "Status",
+    "confirmed": "Confirmed",
+    "pending": "Pending",
+    "failed": "Failed",
+    "network": "Network",
+    "blockHeight": "Block Height",
+    "to": "To",
+    "from": "From"
+  },
+  "trade": {
+    "searchingRate": "Searching for best rate...",
+    "previewTrade": "Preview Trade",
+    "rate": "Rate",
+    "minerFee": "Miner Fee",
+    "shapeshiftFee": "ShapeShift Fee",
+    "confirmTrade": "Confirm Trade",
+    "confirmAndTrade": "Confirm and Trade",
+    "complete": "Trade Complete",
+    "approveAsset": "Approve interacting with %{symbol}",
+    "needPermission": "We need your permission to use your %{symbol}",
+    "whyNeedThis": "Why do I need to do this?",
+    "estimatedGasFee": "Estimated Gas Fee",
+    "approvingAsset": "Approving %{symbol}",
+    "viewTransaction": "View Transaction",
+    "tooltip": {
+      "rate": "This is the rate",
+      "minerFee": "This is the Miner Fee",
+      "shapeshiftFee": "This is the Shapeshift Fee"
```

Comment:
```suggestion
      "shapeshiftFee": "This is the ShapeShift fee"
```

Reply:
why would fee be capitalized? also Github is being jank and making this look like there is a massive diff even though only 4 phrases were added.